commons/connector/health/health_activity.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "org.radarcns.connector.health",
+  "name": "HKWorkoutTypeIdentifier_HKCategoryTypeIdentifierSleepAnalysis",
+  "type": "record",
+  "doc": "Activity , unit is activity type.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Apple Healthkit ."
+    },
+    {
+      "name": "timeInterval",
+      "type": "int",
+      "doc": "Chronological window size (s)."
+    },
+    {
+      "name": "activity",
+      "type": "int",
+      "doc": "Activity taken in this period."
+    }
```

Comment:
Please use an [Avro enum](https://avro.apache.org/docs/current/spec.html#Enums) here to list the possible activities. First in the list can be `UNKNOWN`, and please also include `OTHER`, where `UNKNOWN` mens you do not know what the activity was, and `OTHER` means the activity was not included in the enum.